rrmngmnt/host.py

```diff
@@ -252,11 +253,26 @@ def copy_to(self, resource, src, dst):
         :param dst: path to destination
         :type dst: str
         """
+        file_permissions = resource.os.get_file_permissions(path=src)
+        file_owner_user, file_owner_group = resource.os.get_file_owner(
+            path=src
+        )
         with resource.executor().session() as resource_session:
             with self.executor().session() as host_session:
                 with resource_session.open_file(src, 'rb') as resource_file:
                     with host_session.open_file(dst, 'wb') as host_file:
                         host_file.write(resource_file.read())
+        self.fs.chmod(path=dst, mode=file_permissions)
+        if (
+            self.os.user_exists(user_name=file_owner_user) or
+            self.os.group_exists(group_name=file_owner_group)
+        ):
+            if self.os.user_exists(user_name=file_owner_user):
+                self.fs.chown(path=dst, username=file_owner_user, groupname="")
```

Comment:
I was thinking about it little bit and I think, it is not good idea to try set same ownership of file as it was on source machine.

so I suggest to do it like this:
```python
def copy_to(self, resource, src, dst, mode=None, ownership=None):
    # ...
    if mode:
        chmod(file, mode)
    if ownership:
        if not instance(ownership, tuple):
            raise TypeError("...")
        chown(file, *ownership)
    # ...
```
what do you think ?